src/lava/lib/optimization/solvers/generic/solver.py

```diff
@@ -17,9 +18,13 @@
 from lava.magma.core.sync.protocol import AbstractSyncProtocol
 from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
 from lava.proc.dense.models import PyDenseModelFloat
+from lava.proc.dense.ncmodels import NcModelDense
 from lava.proc.dense.process import Dense
 from lava.proc.read_gate.models import ReadGatePyModel
+from lava.proc.read_gate.ncmodels import ReadGateCModel
 from lava.proc.read_gate.process import ReadGate
+from lava.proc.scif.ncmodels import NcModelQuboScif
+from lava.proc.scif.process import QuboScif

```

Comment:
I guess this path will not be correct anymore once the relocation paths have been merged? Please check also for the other procs.